mithril-common/src/messages/cardano_transaction_snapshot_list.rs

```diff
@@ -62,19 +65,15 @@ mod tests {
 
     // Test the retro compatibility with possible future upgrades.
     #[test]
-    fn test_v1() {
+    fn test_v2() {
```

Comment:
Same comment as above:
```suggestion
    fn test_v1() {
```